scalableminds/webknossos
# Improve convert to hybrid workflow

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create a volume-only annotation
- Annotation Type: X is removed from info tab
- the layers tab now displays an Add Skeleton Annotation Layer button with the old convert to hybrid functionality
- segmentation layers which were not previously editable now show an (un)lock icon button which shortcuts to the "add volume layer" modal with the layer being preselected
- should be disallowed for tasks

### Issues:
- fixes #6209

------
(Please delete unneeded items, merge only when none are left open)
